src/coredump/_UCD_access_reg_linux.c

```diff
@@ -57,6 +57,9 @@ _UCD_access_reg (unw_addr_space_t as,
 #elif defined(UNW_TARGET_S390X)
   if (regnum > UNW_S390X_R15)
     goto badreg;
+#elif defined(UNW_TARGET_IA64)
```

Comment:
Why only `defined(UNW_TARGET_IA64)` though?

[That patch in Gentoo](https://gitweb.gentoo.org/repo/gentoo.git/tree/sys-libs/libunwind/files/libunwind-1.2-coredump-regs.patch?id=e01ced2da69c0d913e4c91467a0ca5fc3615edcd) has:
```
#elif defined(UNW_TARGET_IA64) || defined(UNW_TARGET_HPPA) || defined(UNW_TARGET_PPC32) || defined(UNW_TARGET_PPC64)
```

Reply:
I extracted the patch from Debian which pointed to Gentoo issue. Makes sense to add those conditions.

Reply:
Fixed in #269.